troubleshooting/commonly-occurring-error-conditions.adoc

```diff
@@ -0,0 +1,37 @@
+:_content-type: ASSEMBLY
+[id="commonly-occurring-error-conditions"]
+= Commonly occurring error conditions
+include::modules/common-attributes.adoc[]
+:context: error-conditions
+
+toc::[]
+
+
+[role="_abstract"]
+The vast majority of errors occur during Collector startup, when the Collector configures itself, finds or downloads an appropriate kernel driver for the system, and then starts collecting events. 
+The following diagram describes the main parts of the startup process:
+
+<Diagram>
```

Comment:
Since we don't have a diagram, we can remove this part. We can revisit the content again when we have the diagram.

Reply:
ack